masterlist.yaml

```
@@ -5407,7 +5410,7 @@ plugins:
         condition: 'active("EconomyOverhaulandSpeechcraftImprovements.esp") and not active("Pan NPCs DB - Economy Overhaul Speechcraft Improvements.esp")'
       - <<: *patchProvided
         subs: [ 'AI Overhaul' ]
```

Since the patch is hosted on another page, the message has to be updated accordingly:
```suggestion
      - <<: *patch3rdParty
        subs:
          - 'AI Overhaul'
          - '[Pandorable''s NPCs and AI Overhaul SSE Patches](https://www.nexusmods.com/skyrimspecialedition/mods/60566/)'
```